src/GHC/RTS/Events.hs

```diff
@@ -621,7 +621,10 @@ buildEventType (EventType num dsc msz) =
   <> maybe "variable" TB.decimal msz <> ")"
 
 -- | Pretty prints an 'Event', with clean handling for 'UnknownEvent'
-ppEvent :: IntMap EventType -> Event -> String
+ppEvent 
+  :: IntMap EventType -- ^ Look up @'UnknownEvent'.'ref'@ to find a suitable description.
```

Comment:
What does this Haddock syntax do, visually and link-wise?

Reply:
![image](https://user-images.githubusercontent.com/75746/153708636-98f41eca-5745-4892-bcc4-eab83e0454b7.png)

Reply:
linking to where you would expect

Reply:
TIL. And what happens when you remove the '@'?